docs/features/features-walk-through/moderation.md

```diff
@@ -0,0 +1,16 @@
+# Administration panel
+
+The administration panel is a feature in InvenioRDM introduced in v10 that provides a graphical user interface for managing users and records. It is designed to be used by administrators and superusers to moderate the content of the repository.
+For more technical details you can read the [developer guide to the InvenioRDM administration panel](../../develop/topics/administration_panel.md), detailing its programmatic interface and usage.
```

Comment:
Note to self: will have to change this link in https://github.com/inveniosoftware/docs-invenio-rdm/pull/760